inductiva/resources/machines.py

```diff
@@ -27,13 +27,15 @@ def __init__(
             more information about machine types.
             num_machines: The number of virtual machines to launch.
             spot: Whether to use spot machines.
-            disk_size_gb: The size of the disk in GB, recommended min. is 20 GB.
+            disk_size_gb: The size of the disk in GB. Minimum is 20 GB.
```

Comment:
Make this more generally configurable, since at least for now when we add more docker images to this will always need to be configured.